mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -33,157 +36,135 @@ interface PasscodePromptProps {
-          dark={tailwind('text-white')}
-          light={tailwind('text-black')}
-          iconType='MaterialIcons'
-          name='close'
-          size={26}
-        />
-      </ThemedTouchableOpacity>
-
-      <ThemedView
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
-        style={tailwind('w-full flex-1 flex-col')}
+        <ThemedViewV2
+          light={tailwind('border-mono-light-v2-900')}
+          dark={tailwind('border-mono-dark-v2-900')}
+          style={[tailwind('w-6 h-6 flex justify-center items-center rounded-full'), { borderWidth: 1.5 }]}
+        >
+          <ThemedIcon
+            dark={tailwind('text-mono-dark-v2-900')}
+            light={tailwind('text-mono-light-v2-900')}
+            iconType='MaterialIcons'
+            name='close'
+            size={16}
+          />
+        </ThemedViewV2>
+      </ThemedTouchableOpacityV2>
+      <ThemedViewV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
+        style={tailwind('w-full flex-1 flex-col px-5')}
       >
-
-        {props.status === TransactionStatus.AUTHORIZED
-          ? (
-            <SuccessMessage
-              message={props.transaction === undefined ? props.grantedAccessMessage : props.authorizedTransactionMessage}
-            />
-          )
-          : (
-            <ThemedView
-              light={tailwind('bg-white')}
-            >
-              <ThemedText
-                style={tailwind('text-center text-xl font-bold px-1')}
-              >
-                {props.title}
-              </ThemedText>
-
-              {props.additionalMessage !== undefined && (
-                <View style={tailwind('text-sm text-center')}>
-                  <ThemedText
-                    testID='txn_authorization_message'
-                    dark={tailwind('text-gray-400')}
-                    light={tailwind('text-gray-500')}
-                    style={tailwind('pt-4 text-base text-center')}
-                  >
-                    {props.additionalMessage}
-                  </ThemedText>
-                </View>
-              )}
-              {props.additionalMessageUrl !== undefined && (
-                <View style={tailwind('text-sm text-center')}>
-                  <ThemedText
-                    testID='txn_authorization_message'
-                    dark={tailwind('text-gray-400')}
-                    light={tailwind('text-gray-500')}
-                    style={tailwind('p-2 text-sm text-center mb-4')}
-                  >
-                    {props.additionalMessageUrl}
-                  </ThemedText>
-                </View>
-              )}
-
-              <View style={tailwind('px-8 text-sm text-center mb-6')}>
-                <ThemedText
+        <ThemedTextV2
+          style={tailwind('text-center font-normal-v2 pt-3.5', { 'mb-16': props.status !== TransactionStatus.SIGNING && props.status !== TransactionStatus.AUTHORIZED })}
```

Comment:
let's add a `px-10`here, this is not in figma but designer wants to avoid having one word at the second sentence, e.g.:
![image](https://user-images.githubusercontent.com/40191153/178479576-b95446fc-90c7-4bfb-a78c-0cf7f88f84b5.png)

Reply:
and change top padding to 6px